synapse/rest/admin/users.py

```diff
@@ -832,8 +833,33 @@ async def on_GET(
                 errcode=Codes.INVALID_PARAM,
             )
 
+        # If neither `order_by` nor `dir` is set, set the default order
+        # to newest media is on top for backward compatibility.
+        if b"order_by" not in request.args and b"dir" not in request.args:
+            order_by = MediaSortOrder.CREATED_TS.value
+            direction = "b"
```

Comment:
Isn't this identical to providing the `default` key below?

Reply:
This is for the special case of backward compatibility of this API. In case of no request parameter there is a special value of default: direction is `b`. In a normal case the direction is `f`.
It was a request: https://github.com/matrix-org/synapse/pull/8978#discussion_r574612911

Reply:
Ah, right! I missed that the direction was opposite the default. Good catch!